dashboard/test/factories.rb

```diff
@@ -26,6 +26,11 @@
     factory :student do
       user_type User::TYPE_STUDENT
     end
+
+    factory :young_student do
+      user_type User::TYPE_STUDENT
+      birthday Date.new(2009, 03, 14)
```

Comment:
Is there a better approach? I think I originally had it as 2014, but got a complaint that this wasn't a legit age. I guess I could probably do something like Date.today - Date.years(10)